mllib/src/test/scala/org/apache/spark/mllib/clustering/PowerIterationClusteringSuite.scala

```diff
@@ -82,7 +82,7 @@ class PowerIterationClusteringSuite extends SparkFunSuite with MLlibTestSparkCon
     // Generate two circles following the example in the PIC paper.
     val r1 = 1.0
     val n1 = 10
-    val r2 = 4.0
+    val r2 = 14.0
```

Comment:
Per the previous discussion, can this change be reverted if n values are increased?

Reply:
done